packages/ember-glimmer/lib/environment.js

```diff
@@ -69,6 +69,9 @@ export default class Environment extends GlimmerEnvironment {
     super(...arguments);
     this.owner = owner;
 
+    // can be removed once https://github.com/tildeio/glimmer/pull/305 lands
+    this.destroyedComponents = undefined;
```

Comment:
Right, I'd rather have a small kludge here in Ember for beta.1 and fix upstream in that other PR for beta.2.